src/sezzle-checkout-installment-widget/sezzle-checkout-installment-widget.js

```diff
@@ -0,0 +1,380 @@
+		modalOverlay.appendChild(modalContent);
+
+		// creates the close modal button
+		let closeModal = document.createElement('button');
+		closeModal.className = 'close-sezzle-modal';
+		closeModal.role = 'button';
+		closeModal.type = 'button';
+		closeModal.title = 'Close Modal';
+		closeModal.innerText = 'X';
+		modalContent.appendChild(closeModal);
+
+		// creates the Sezzle logo
+		let sezzleLogo = document.createElement('div');
+		sezzleLogo.className = 'sezzle-modal-logo';
+		sezzleLogo.title = 'Sezzle Logo';
+		modalContent.appendChild(sezzleLogo);
+
+		// creates the modal title
+		let modalTitle = document.createElement('h4');
+		modalTitle.className = 'sezzle-modal-title';
+		modalContent.appendChild(modalTitle);
+
+		// creates the description container
+		let overview = document.createElement('div');
+		overview.className = 'sezzle-modal-overview';
+		modalContent.appendChild(overview);
+
+		// creates the first overview paragraph
+		let firstParagraph = document.createElement('p');
+		overview.appendChild(firstParagraph);
+
+		// creates the second overview paragraph
+		let secondParagraph = document.createElement('p');
+		overview.appendChild(secondParagraph);
+
+		// creates the modal pie graphic
+		let modalPie = document.createElement('div');
+		modalPie.className = 'sezzle-modal-payment-pie';
+		modalContent.appendChild(modalPie);
+
+		// creates the installment schedule container
+		let percentages = document.createElement('div');
+		percentages.className = 'sezzle-modal-payment-percent';
+		modalContent.appendChild(percentages);
+
+		// creates each percentage
+		for(let i = 0; i < 4; i++){
+			let percent = document.createElement('span');
+			percent.innerText = '25%';
+			percentages.appendChild(percent);
+		}
+
+		// creates the installment schedule container
+		let sampleSchedule = document.createElement('div');
+		sampleSchedule.className = 'sezzle-modal-payment-schedule';
+		modalContent.appendChild(sampleSchedule);
+
+		// creates each installment
+		for(let i = 0; i < 4; i++){
+			let payment = document.createElement('span');
+			if (i === 0){
+				payment.innerHTML = language === 'fr' ? 'aujourd\'hui' : 'today';
+			} else {
+				payment.innerHTML = (language === 'fr' ? 'semaine ' : 'week ' ) + i*2;
+			}
+			sampleSchedule.appendChild(payment);
+		}
+
+		// handles translations
+		if(language === 'fr'){
+			installmentWidget.innerHTML = '4 paiement sans inte&#769;re&#770;ts sur 6 semaines';
+			modalTitle.innerHTML = 'Comment &#231;a fonctionne';
+			firstParagraph.innerHTML = 'R&#233;partir le montant de votre commande en 4 versements sans int&#233;r&#234;ts &#233;tal&#233;s sur 6 semaines, pas de frais si vous payez aux dates pr&#233;vues, et pas d\'impact sur votre cote de cr&#233;dit.';
+			secondParagraph.innerHTML = 'Apr&#232;s avoir cliqu&#233; sur &#171;Terminer la commande&#187; sur ce site, vous serez redirig&#233; vers Sezzle pour finaliser votre achat en toute s&#233;curit&#233;.'
```

Comment:
Après avoir cliqué sur « Terminer la commande » sur ce site, vous serez redirigé(e) vers Sezzle pour finaliser votre achat en toute sécurité.

Reply:
The space after the (weird) opening quote mark and before the closing quote mark is supposed to be a non-breaking space. In HTML I would use `&nbsp;` ...but your codes are even weirder than that, so I don't know 😋